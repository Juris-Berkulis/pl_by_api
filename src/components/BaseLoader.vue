<script setup lang="ts">
</script>

<template>
<div class="h-full w-full flex justify-center items-center">
    <div class="animate-spin h-12 w-12 border-8 border-sky-200 border-r-transparent rounded-full"></div>
</div>
</template>

<style scoped>
</style>
