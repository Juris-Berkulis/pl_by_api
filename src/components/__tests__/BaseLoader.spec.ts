import { mount } from "@vue/test-utils";
import { describe, expect, it } from "vitest";
import BaseLoader from "../BaseLoader.vue";

describe('BaseLoader', () => {
    const wrapper = mount(BaseLoader);

    it('Snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
