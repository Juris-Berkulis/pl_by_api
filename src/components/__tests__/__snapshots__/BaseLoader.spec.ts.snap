// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`BaseLoader > Snapshot 1`] = `
"<div class=\\"h-full w-full flex justify-center items-center\\">
  <div class=\\"animate-spin h-12 w-12 border-8 border-sky-200 border-r-transparent rounded-full\\"></div>
</div>"
`;
